cluster-operator/src/main/resources/kafka-2.2.1-config-model.json

```diff
@@ -1,825 +0,0 @@
-{
```

Comment:
The `cluster-operator/src/main/resources/kafka-*-config-model.json` files are generated and should be part of the code base. I need to have a look wh did it got deleted. Could you please restore them int he PR in the mean time?

@tombentley Why are these actually in the code base? Isn't it enough to have these in the binaries?

Reply:
the mentioned files were added back.